src/Widgets/NotificationEntry.vala

```diff
@@ -196,10 +196,9 @@ public class Notifications.NotificationEntry : Gtk.ListBoxRow {
             return Gdk.EVENT_STOP;
         });
 
-        notification.time_changed.connect ((timestamp) => {
-            time_label.label = Granite.DateTime.get_relative_datetime (timestamp);
-
-            return active;
+        timeout_id = Timeout.add_seconds_full (Priority.DEFAULT, 60, () => {
+            time_label.label = Granite.DateTime.get_relative_datetime (notification.timestamp);
+            return true;
```

Comment:
Forget about my previous comment, I haven't understood the function, this would make it more obvious:
```suggestion
            return GLib.Source.CONTINUE;
```